docs/guides/connecting_to_your_data/datasource_configuration/components/_section_intro.mdx

```diff
@@ -1,4 +1,4 @@
 <!--- props.backend: The name of the backend used by the Datasource's Execution Engine. -->
 <p>
-This guide will walk you through the process of configuring a {props.backend} Datasource from scratch, verifying that your configuration is valid, and adding it to your Data Context.  By the end of this guide you will have a {props.backend} Datasource which you can use in future workflows for creating Expectations and validating data.
+This guide will walk you through the process of configuring a {props.backend} Datasource from scratch, verifying that your configuration is valid, and adding it to your Data Context.  By the end of this guide you will have a {props.backend} Datasource which you can use in future workflows for creating Expectations and Validating data.
```

Comment:
It's capitalized because we have a technical term entry for "The act of applying an Expectation Suite to a Batch" (the entry is technically for the word "Validation", but I've been applying it to all variations of the Validate verb as well) and we always capitalize our technical terms.  That will be more evident when I tag all of our technical terms in a future PR.

Reply:
And I'm daft.  You're pointing out that it wasn't capitalized.  Yes, that is a mistake on my end; I'll push an update forthwith.